common/api/components-react.api.md

```diff
@@ -260,6 +260,7 @@ export interface CommonPropertyGridProps extends CommonProps {
     onPropertyUpdated?: (args: PropertyUpdatedArgs, category: PropertyCategory) => Promise<boolean>;
     orientation?: Orientation;
     propertyValueRendererManager?: PropertyValueRendererManager;
+    showOnlyEditor?: (editorName?: string) => boolean;
```

Comment:
I think `showOnlyEditor` should get whole `PropertyRecord` instead of only editor name.

Reply:
Adjusted.